src/libre/ac.c

```diff
@@ -25,6 +26,9 @@ struct trie_state {
 	unsigned int index;
 	unsigned int output:1;
 	unsigned int have_st:1;
+
+	/* use a state set as an endid set */
+	struct state_set *endids;
```

Comment:
could you put this above the `fsm_state_t` please? i think it'll help keep things grouped for struck packing

Reply:
Sure, done in 8b693385.